phys2bids/tests/test_integration.py

```diff
@@ -32,14 +33,14 @@ def test_logger(multifreq_lab_file):
     test_chtrig = 3
     test_ntp = 1
     test_outdir = test_path
-
+    extra_dir = test_path + '/bids_ignore/'
```

Comment:
I'm writing it here, but can you check it in across tests?

As a general thing, you could make paths with `os.path.join(test_path, 'bids_ignore')` rather that with string unions.
It's better for compatibility since it doesn't assume `/` to be the separator (although probably azure would not break due to the same function being used later).

If you're using frequently this variable with the same initialisation you could make it a constant (just add it after the `import` section in capital letter). Then you can import it to other testing files (or from other testing files to this one).

Reply:
(PS: Thank you for the `'`!)

Reply:
Looks like we both asked for the same changes haha